### PR TITLE
Detr demos and tutorial

### PR DESCRIPTION
**Pull request by jelledouwe**
Dear reviewers,

Here are some minor changes for the DetrLearner. One of the argument names is changed to make it more informative and a utility method for drawing results is add. Furthermore, demos and an inference tutorial are created and the documentation is updated accordingly.

### CODE REVIEW

File: docs/reference/detr.md [hunk 1]
```diff
@@ -72,34 +73,34 @@ train_annotations_file, val_images_folder, val_annotations_file)
 This method is used for training the algorithm on a train dataset and validating on a val dataset.
 Returns a dictionary containing stats regarding the last evaluation ran.
 Parameters:
```
Comment (ad-daniel) [inline docs/reference/detr.md:75]:
```suggestion

Parameters:
```

File: docs/reference/detr.md [hunk 2]
```diff
@@ -111,20 +112,20 @@ DetrLearner.eval(self, dataset, images_folder, annotations_folder, annotations_f
 This method is used to evaluate a trained model on an evaluation dataset.
 Returns a dictionary containing stats regarding evaluation.
 Parameters:
```
Comment (ad-daniel) [inline docs/reference/detr.md:114]:
```suggestion

Parameters:
```

File: projects/perception/object_detection_2d/detr/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# DETR Demos
+
+This folder contains minimal code usage examples that showcase the basic functionality of the DetrLearner provided by 
+OpenDR.
```
Comment (ad-daniel) [inline projects/perception/object_detection_2d/detr/README.md:4]:
```suggestion
This folder contains minimal code usage examples that showcase the basic functionality of the DetrLearner provided by OpenDR. 
```

Comment (ad-daniel) [reply projects/perception/object_detection_2d/detr/README.md:4]: one sentence per line

File: src/opendr/perception/object_detection_2d/detr/README.md [hunk 4]
```diff
@@ -19,3 +19,6 @@ The modifications are:
   DETR was originally licensed under the Apache 2.0 [license](https://github.com/facebookresearch/detr/blob/master/LICENSE).
 
   The modifications are also licensed under the Apache 2.0 license by OpenDR European Project.
+
+Also, code from [here](https://stackoverflow.com/questions/60674501/how-to-make-black-background-in-cv2-puttext-with-python-opencv)
+is used in the [drawing utility function](algorithm/util/draw.py).
```
Comment (ad-daniel) [inline src/opendr/perception/object_detection_2d/detr/README.md:24]:
```suggestion
Also, code from [here](https://stackoverflow.com/questions/60674501/how-to-make-black-background-in-cv2-puttext-with-python-opencv) is used in the [drawing utility function](algorithm/util/draw.py).
```

**Review by ad-daniel — CHANGES_REQUESTED** [review]
Very good, thank you!

**Review by ad-daniel — APPROVED** [review]
Thank you!

**Review by passalis — APPROVED** [review]
Nice work! Thank you!

**Pull request merged**